README.md

```diff
@@ -9,13 +9,14 @@ A very simple and intuitive mysql client wrapper for sqlalchemy.
 
 ## Installation
 
-
 - Run the command to install: `pip install nmi_mysql`
-- Make sure you install [`sqlalchemy`](http://www.sqlalchemy.org/)
+- Make sure you install [`sqlalchemy`](http://www.sqlalchemy.org/) and [`pymysql`](http://www.pymysql.org/)
```

Comment:
Please indicate what we use pymysql for, and include the reference I found as reference of making the decision vs mysql-connector